performance/tests/Button.test.js

```diff
@@ -0,0 +1,100 @@
+const puppeteer = require('puppeteer');
+const {FPS, Mount, Update} = require('../TraceModel');
+const {getFileName} = require('../utils');
+const TestResults = require('../TestResults');
+
+describe('Button', () => {
+	describe('click', () => {
+		it('increment', async () => {
+			const filename = getFileName('Button');
+			const button = '[class^="Button"]';
+
+			const browser = await puppeteer.launch({headless: true});
+			const page = await browser.newPage();
+			await page.setViewport({
+				width: 1920,
+				height: 1080
+			});
+
+			await page.goto('http://localhost:8080/button');
+			await page.tracing.start({path: filename, screenshots: false});
+			await page.waitFor(500);
+			await page.click(button);
+			await page.waitFor(200);
+			await page.click(button);
+			await page.waitFor(200);
+			await page.click(button);
+			await page.waitFor(200);
+			await page.click(button);
+			await page.waitFor(200);
+
+			await page.tracing.stop();
+			await browser.close();
+
+			const actualFPS = FPS(filename);
+			TestResults.addResult({component: 'Button', type: 'Frames Per Second', actualValue: actualFPS});
+
+			const actualUpdateTime = Update(filename, 'Changeable');
+			TestResults.addResult({component: 'Button', type: 'Update', actualValue: actualUpdateTime});
+
+		});
+	});
+
+	describe('keypress', () => {
+		it('increment', async () => {
+			const filename = getFileName('Button');
+
+			const browser = await puppeteer.launch({headless: true});
+			const page = await browser.newPage();
+			await page.setViewport({
+				width: 1920,
+				height: 1080
+			});
+
+			await page.goto('http://localhost:8080/picker');
+			await page.tracing.start({path: filename, screenshots: false});
+			await page.waitFor(500);
+			await page.keyboard.press('Enter');
```

Comment:
We also may want to focus before we use enter. You can check my Scroller PR for this code.